website/public/js/services/guideServices.js

```diff
@@ -239,6 +239,7 @@ function($rootScope, User, $timeout, $state) {
     if (!updateFn) return; // only run after user has been wrapped
     watcher(); // deregister watcher
     if (window.env.IS_MOBILE) return; // Don't show tour immediately on mobile devices
+    if (User.user.items.gear.owned['weapon_warrior_0']) return;//If the user has the training sword, then don't show the intro tour
```

Comment:
@crookedneighbor If we set the flag after checking if the user is on the mobile device, then we should not collapse the if statement, correct?

@Alys would you like me to change my current comment, "//If the user has the training sword, then don't show the intro tour", to your new text?